racadm.py

```diff
@@ -133,71 +137,74 @@ def logout(self):
         Close the session so we are not stealling spaces
         '''
         #this dosn't work because 'Closing the executing session is not permitted.'
```

Comment:
Do you know how to work around the "Closing th executing session is not permitted."-error? I'm writing a script for configuring servers using the iDrac and cannot find anything online as to how to fix this.